src/mha/problem_description.cpp

```diff
@@ -37,11 +37,20 @@ NetworkConfig ProblemDescription::MakeNetworkConfig() const
 {
     std::ostringstream ss;
 
-    ss << "mhafwd-";
+    ss << "mha";
 
     if(isForward)
     {
-        // TODO Implement
+        ss << "fwd-";
+        for(auto s : mhaInputDescsForward.oDesc.GetLengths())
+        {
+            ss << s << "x";
```

Comment:
Is the extra x on the end of the Lengths / Strides for the Type?
ex. mhafwd-1x1x1xFP32

Reply:
Yep. It's just a DB key.